src/components/ArticleCards.jsx

```diff
@@ -1,15 +1,22 @@
+import { Link } from "react-router-dom"
 function ArticleCards({ article }) {
-
-    return (
-      <>
-        <h3>{article.title}</h3>
-        <img className="article-image" src={article.article_img_url} alt={article.title} />
+  return (
+    <>
```

Comment:
There is no need to wrap a section in a fragment, the section appears to contain everything that is needed so the fragment is redundant.